reference/docs-conceptual/learn/shell/using-light-theme.md

```diff
@@ -0,0 +1,138 @@
+DefaultTokenColor                      : "`e[37m"
+EmphasisColor                          : "`e[96m"
+ErrorColor                             : "`e[91m"
+InlinePredictionColor                  : "`e[38;5;238m"
+KeywordColor                           : "`e[92m"
+ListPredictionColor                    : "`e[33m"
+ListPredictionSelectedColor            : "`e[48;5;238m"
+MemberColor                            : "`e[97m"
+NumberColor                            : "`e[97m"
+OperatorColor                          : "`e[90m"
+ParameterColor                         : "`e[90m"
+SelectionColor                         : "`e[30;47m"
+StringColor                            : "`e[36m"
+TypeColor                              : "`e[37m"
+VariableColor                          : "`e[92m"
+```
+
+The color settings are stored as strings containing ANSI escape sequences that change the color in
+your terminal. Using the `Set-PSReadLineOption` cmdlet you can change the colors to values that work
+better for a light-colored background.
+
+## Defining colors for a light theme
+
+The original PowerShell ISE can be configured to use a light theme for both the editor and console
+panes. You can also view and change the colors that the ISE uses for various syntax and output
+types. You can use these color choices to define a similar theme for **PSReadLine**.
+
+The following hashtable defines colors for **PSReadLine** that mimic the colors in the PowerShell
+ISE.
+
+```powershell
+$ISETheme = @{
+    Command                  = $PSStyle.Foreground.FromRGB(0x0000FF)
+    Comment                  = $PSStyle.Foreground.FromRGB(0x006400)
+    ContinuationPrompt       = $PSStyle.Foreground.FromRGB(0x0000FF)
+    Default                  = $PSStyle.Foreground.FromRGB(0x0000FF)
+    Emphasis                 = $PSStyle.Foreground.FromRGB(0x287BF0)
+    Error                    = $PSStyle.Foreground.FromRGB(0xE50000)
+    InlinePrediction         = $PSStyle.Foreground.FromRGB(0x93A1A1)
+    Keyword                  = $PSStyle.Foreground.FromRGB(0x00008b)
+    ListPrediction           = $PSStyle.Foreground.FromRGB(0x06DE00)
+    Member                   = $PSStyle.Foreground.FromRGB(0x000000)
+    Number                   = $PSStyle.Foreground.FromRGB(0x800080)
+    Operator                 = $PSStyle.Foreground.FromRGB(0x757575)
+    Parameter                = $PSStyle.Foreground.FromRGB(0x000080)
+    String                   = $PSStyle.Foreground.FromRGB(0x8b0000)
+    Type                     = $PSStyle.Foreground.FromRGB(0x008080)
+    Variable                 = $PSStyle.Foreground.FromRGB(0xff4500)
+    ListPredictionSelected   = $PSStyle.Background.FromRGB(0x93A1A1)
+    Selection                = $PSStyle.Background.FromRGB(0x00BFFF)
+}
+```
+
+In PowerShell 7.2 and higher you can use the `FromRGB()` method of `$PSStyle` to create the ANSI
+escape sequences for the colors you want.
+
+For more information about `PSStyle`, see [about_ANSI_Terminals][01].
```

Comment:
Nit, and possibly out of scope for this article, but:

With `about_ANSI_Terminals` only existing for 7.2+, it might be useful to define a conceptual doc (or maybe just an aside?) for ANSI terminals that talks about the `${esc}` usage in the _Customizing your shell environment_ article and the cross-version output handling.

If this wasn't for profiles, I wouldn't think about it as much, but I know many people (including myself) have one profile that they use for multiple versions/platforms.

Reply:
Good idea for a separate PR